docker/data-science/notebooks/ch06/2.related-keywords-from-signals.ipynb

```diff
@@ -489,6 +494,7 @@
       "+------------+-------+\n",
       "|     keyword|n_users|\n",
       "+------------+-------+\n",
+      "|   star wars|   7564|\n",
```

Comment:
I think this cell was run AFTER the generation of the 5000 fake signals for the spam example in listing 8.3.  I thought those spam signals were only put in a spam signals collection to avoid this (in ch8), but it seems that they are actually added to the main signals collection and then only aggregated into a separate spam collection.

We probably need to copy the signals collection to a "signals_with_spam" collection first before adding the spam signals to ensure no side effects like this with other chapters.